components/tor/onion_location_navigation_throttle.cc

```diff
@@ -71,15 +72,16 @@ OnionLocationNavigationThrottle::WillProcessResponse() {
   // The webpage defining the Onion-Location header must not be an onionsite.
   // https://gitlab.torproject.org/tpo/applications/tor-browser-spec/-/raw/HEAD/proposals/100-onion-location-header.txt
   if (headers && GetOnionLocation(headers, &onion_location) &&
-      !navigation_handle()->GetURL().DomainIs("onion") &&
+      !navigation_handle()->GetURL().DomainIs(kOnionDomain) &&
```

Comment:
Not sure whether this would be usable here, but we've also got an `IsOnion()` in https://github.com/brave/brave-core/blob/7fb458e780ab6165f4a44402b4fe59cbd8647ff4/chromium_src/net/base/url_util.cc#L42-L44

Reply:
We can all use that util `IsOnion` with `DomainIs` which is more efficient
https://source.chromium.org/chromium/chromium/src/+/main:url/gurl.h;l=409-412;drc=d54f6988e3ac778735a4082ab9bbe2a178e40d74